app/models/euro.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Euro < Account
```

Comment:
IMHO we should allow a user to create own currencies and do not hardcode them

Reply:
okey, i will check how to realize that
and i am sorry for slow response, notifications broken on my phone(

Reply:
I am going to use this [](https://github.com/RubyMoney/money) gem for exchanging money.
What do you thing about it?

Reply:
Decent solution. Have nothing against it.